Find a functionally equivalent code:android.print.PrintManager.createPrinterDiscoverySession:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * @hide
 */

@@ -1,3 +1,7 @@
 {
+    if (mService == null) {
+        Log.w(LOG_TAG, "Feature android.software.print not available");
+        return null;
+    }
     return new PrinterDiscoverySession(mService, mContext, mUserId);
 }

